Jenkinsfile

```diff
@@ -40,19 +40,23 @@ def buildAndTest(nodeLabel, target, rakeEnv, encrypted, testIam='no') {
 
         // Build and test
         try {
-            def envVariables = []
+            def envVariables = ["TEST_COUCH_LOGGING_LEVEL=31"]
```

Comment:
Is this extra logging left on intentionally, or is the plan to enable it as needed?

Reply:
It was to help debug some test failures, but I wonder whether we should leave it in? It's frustrating to have a test failure and not have detailed test logs, especially if it's a transient failure.

The only downside I can is the storage requirements but the log files are only a few megabytes.

Reply:
I'm fine with that, just checking it wasn't an oversight